### ISSUE TITLE
Kosovo Data

### ISSUE THREAD

**Issue by akarlinsky** [akarlinsky/world_mortality#14]
Kosovo's NSO (KAS or ASK) just released a report on total deaths in [2020](https://ask.rks-gov.net/media/6093/statistikat-e-vdekjeve-2020.pdf)
The total is given at 14,489 deaths - which is 30% higher than the sum of our monthly deaths for 2020 (11,108). 
The 2020 report does not give figures by month and states (translated by Google-Translate):

_Kosovo Agency of Statistics (KAS), respectively the Department of Social Statistics (DSS) / Division of Population Statistics, publishes data on death statistics in Kosovo and Outside Kosovo for 2020.
Cases of deaths occurred in 2020 according to source data available to the Kosovo Agency of Statistics for death statistics are the Registry Books with no. 2004/46 for death.
Reporting units are the Kosovo Civil Registry Offices (Registries).
Kosovo Agency of Statistics (KAS) estimates that the number of forms collected does not represent / cover all cases of deaths occurred and recorded in Kosovo. This is particularly pronounced in the months when COVID-19 infection and mortality rates were highest, where many statistical forms / sheets were not reported to KAS. Therefore, the published data will be preliminary and the source of data is the Kosovo Civil Registry Agency until the final report will be published in the coming months by verifying / supplementing them with detailed data._

The [2019 report on total deaths](https://ask.rks-gov.net/media/5478/statistikat-e-vdekjeve-2019.pdf) numbers are closely aligned to those from the monthly bulletins that we use in WMD (9550 vs 9552).

Should we scale up the monthly number of deaths in Kosovo 2020 by 30%? Or wait for additional updates and incorporate them when they become available?

**Comment by dkobak** [comment]
> Should we scale up the monthly number of deaths in Kosovo 2020 by 30%?

The early 2020 data aligns well with the 2019 and earlier data, so scaling the entire 2020 up by 30% is not an option, imho.

Did we not have a similar issue with quarterly reports from Kosovo not agreeing with the monthly reports? And not only for deaths but also for births? I vaguely remember something like that.

**Comment by akarlinsky** [comment]
The issue you are referring to is the discrepancy between the quarterly bulletins (that list by month) and the newer "monthly vital stats report", which for now are housed at [Preliminary Mortality](https://github.com/akarlinsky/world_mortality/tree/main/preliminary_mortality#kosovo-monthly) to avoid consistency issues. 

Until now, ASK has yet to respond to our query about this discrepancy. 

Another alternative is of course to just wait until ASK releases the final report for deaths in 2020 and then update accordingly.

**Comment by dkobak** [comment]
> Another alternative is of course to just wait until ASK releases the final report for deaths in 2020 and then update accordingly.

But the final report will be yearly, so will likely not contain any monthly data, right? If so, it won't help us...

**Comment by akarlinsky** [comment]
The final final 2020 report should have a monthly breakdown as well, like the 2019 report before it. 
What they released right now is a very trimmed down version with just 2-3 pages.

**Comment by dkobak** [comment]
Oh I see. Then we should simply wait, imho.

**Comment by akarlinsky** [comment]
ASK released the statistical yearbook which has monthly number of deaths per year, except for 2020 where total is given as 12971 (17% higher than our figure). 
![image](https://user-images.githubusercontent.com/48435065/125278101-f42a4d80-e31a-11eb-8ba3-b824348d6d02.png)

https://ask.rks-gov.net/sq/agjencia-e-statistikave-te-kosoves/add-news/vjetari-statistikor-i-republikes-se-kosoves-2021

**Comment by dkobak** [comment]
Not sure what to do about it, but I guess there is nothing we can do at this point unless they either reply to our email query or release an even more updated version of the report with the monthly breakdown.

**Comment by akarlinsky** [comment]
Of course. I'm just putting it here as a matter of record.

**Comment by Nerdm** [comment]
The first number given, 14,489, includes deaths occurring outside [Kosovo](https://ask.rks-gov.net/media/6093/statistikat-e-vdekjeve-2020.pdf) (as Kosovo has a huge diaspora in Europe).
The second number is the total number of deaths INSIDE KOSOVO, that is 14,489 - 1518 (outside of Kosovo) = 12,971.



The data on your screenshot all exclude deaths occuring outside of Kosovo. Maybe, you should use those data in estimation of expected deaths. 
![Screenshot_20210912-201435](https://user-images.githubusercontent.com/90572098/132998359-65aefdd6-a7cd-4a9e-86db-a025d27d5228.png)

**Comment by Nerdm** [comment]
Also, the data on Preliminary Mortality for 2021 include deaths occuring outside of Kosovo. The actual number is lower. 

![Screenshot_20210912-220253](https://user-images.githubusercontent.com/90572098/133001290-96d6879a-d1ec-4d8c-8695-fdcb93b9231e.png)

https://askdata.rks-gov.net/PXWeb/pxweb/en/askdata/askdata__09%20Population__Demography%20and%20Migration__Monthly%20indicators__Births,%20deaths,%20marriages%20and%20divorces%20-%20monthly/tab03.px/?rxid=517b28c3-31b7-41ef-bff5-cc92f18ae656

The March data is not correct: all deaths 1822, outside of Kosovo 233.
PS. In the chart there's another error, it is written "births" instead of deaths.

[Deaths in January 2021](https://ask.rks-gov.net/media/5946/buletini-mujor-janar-2021-popullsiaj.pdf)
[Deaths in February 2021](https://ask.rks-gov.net/media/6247/buletini-mujor-shkurt-2021-popullsiaj.pdf)
[Deaths in March 2021](https://ask.rks-gov.net/media/6246/buletini-mujor-mars-2021-popullsiaj-fial.pdf)
[Deaths in April 2021](https://ask.rks-gov.net/media/6245/buletini-mujor-prill-2021-popullsiaj-1.pdf)
[Deaths in June 2021](https://ask.rks-gov.net/media/6243/buletini-qeshor-2021-shqip.pdf)

Could not find the file for deaths in May 2021, but it is in the chart above.

**Comment by akarlinsky** [comment]
Thanks! I will update Preliminary Mortality accordingly.
But, there's still a large discrepancy between the quarterly reports and the monthly reports/pxstat table generators, right?
For example January 2021 from quarterly report: 1167.
From monthly report: 1509 (total minus "outside Kosovo").

**Comment by Nerdm** [comment]
It is true there are discrepancies between monthly bulletins, statistical yearbook, table generator and the quarterly bulletins. 
The data in the first three are all the same.

![Screenshot_20210913-113410](https://user-images.githubusercontent.com/90572098/133060781-c470434e-2265-43f4-85ad-a2defa61d5e3.png)
![Screenshot_20210913-113440](https://user-images.githubusercontent.com/90572098/133060784-5a14348a-a9b0-451c-9aff-f127bc20c613.png)

The quarterly bulletins data are based, as far as I can tell, on surveys from ASK, while the data on the other three are from Office of Civil Registration (for 2020 at least it is stated as so).

Another important note: there are 10 serb majority municipalities, which due to political situation operate as semi-autonomous municipalities and it is harder to collect data from there by surveys alone. 
And, as per Ministry of Health, those municipalities are not regularly reporting Covid cases and deaths.

My suggestions would be: 
1. The data for deaths in pre-pandemic years to be collected in the statistical yearbook (which you posted in the earlier comment), and this way to compile the expected deaths for 2020 and 2021.
2. As of now, to rely on the total number of deaths inside Kosovo in 2020 (12971), as long as the monthly data are not published yet (based on these numbers),
3. And, as previously mentioned, the data for 2021 to be collected from the monthly bulletins. 

Based on the official covid deaths per months, I think those data align better, especially the 2021 numbers.

PS. All the data to be considered preliminary, as stated by the ASK itself.

**Comment by dkobak** [comment]
Thanks! The problem is that the total number of deaths inside Kosovo in 2020 according to the yearbook is 12,971, but the sum of monthly values that we currently have in our dataset is 11,108. Is there anything we can do about it? Apart from waiting for an updated statistical yearbook?

**Comment by Nerdm** [comment]
I don't have the updated monthly deaths on 2020 now, unfortunately. 
But I'll see if I can get them from the Civil Registry Agency.

**Comment by Nerdm** [comment]
Looks like your data (2015-2020) are based on quarterly bulletins, which differ from those of 1. Yearly deaths bulletins, 2. Statistical yearbook, and 3. Data generated from the table.

**Comment by dkobak** [comment]
Oh right. So there is **non-quarterly-bulletin** information on monthly values (inside/outside Kosovo) for 2021 but not for 2020? Is that right?

**Comment by akarlinsky** [comment]
Yes, we don't have any other source for 2020 monthly other the quarterly bulletins currently.

**Comment by Nerdm** [comment]
Yes, that is right. There aren't monthly data for 2020, exept from quarterly bulletins. 

For 2021 and for pre-pandemic years, there are.

[Pre-pandemic years](https://askdata.rks-gov.net/PXWeb/pxweb/en/askdata/askdata__09%20Population__Demography%20and%20Migration__Yearly%20indicators__Deaths/vdekjet05.px/?rxid=c772099f-e5c2-409f-9faa-cc666573d2a0)
![Screenshot_20210913-143838](https://user-images.githubusercontent.com/90572098/133085062-4985af8a-2924-4940-a128-c57f127a6458.png)

They are the same as those in [statistical yearbook 2021](https://ask.rks-gov.net/sq/agjencia-e-statistikave-te-kosoves/add-news/vjetari-statistikor-i-republikes-se-kosoves-2021)
![125278101-f42a4d80-e31a-11eb-8ba3-b824348d6d02 (1)](https://user-images.githubusercontent.com/90572098/133085461-760a1915-dff5-45ae-9dbf-de96d873e796.png)

[Monthly deaths 2021 inside Kosovo only](https://askdata.rks-gov.net/PXWeb/pxweb/en/askdata/askdata__09%20Population__Demography%20and%20Migration__Monthly%20indicators__Births,%20deaths,%20marriages%20and%20divorces%20-%20monthly/tab03.px/?rxid=c772099f-e5c2-409f-9faa-cc666573d2a0)
![Screenshot_20210913-151227](https://user-images.githubusercontent.com/90572098/133089976-94fc705b-19be-4e2a-aee3-f76a85e99a0f.png)

Also, I've posted the pdf formats above, for 2021 (except for may). 

I hope this helps. 

As for 2020, monthly, I guess we have to wait.

**Comment by Nerdm** [comment]
Another side note: 
Kosovo population (estimation) 2020: 
![Uploading Screenshot_20210913-160805.png…]()

**Comment by Nerdm** [comment]
Another side note: 
Kosovo population (estimation) 2020: 
![Screenshot_20210913-160805](https://user-images.githubusercontent.com/90572098/133098996-2c50b7f2-4cd8-41ad-81fd-8b341d87b3cc.png)

**Comment by dkobak** [comment]
@akarlinsky: So for 2020 there is currently nothing we can do. But:

1. Can we replace the values for 2015-2019 to the ones from the statistical yearbook? 
2. Can we use 2021 values from the monthly bulletins, making sure to subtract "outside of Kosovo" deaths?

**Comment by akarlinsky** [comment]
1. If we do that, our expected figure changes quite substantially, especially for December 2020:
![image](https://user-images.githubusercontent.com/48435065/133241357-953dcded-2d6d-41bd-9693-d2cbee1f5aaa.png).
The "new" expected makes more sense with December similar to January.  But:

2. This will make the data inconsistent. Using the monthly numbers from the quarterly bulletins, we know we are wrong (we always are to some degree), but at least we are consistent. Mixing the quarterly for 2020 and 2015-2019 and 2021 from the monthly makes up inconsistent by definition. 

All in all, the effect is the following: 
Using current data: 2020 excess (March to December): 1740.
Total excess (March 2020 to June 2021): 3840.

With the data mix: 2020 excess (March to December): 1492.
Total excess (March 2020 to June 2021): 4677.

**Comment by Nerdm** [comment]
@akarlinsky what if you increase the monthly data for 2020 by 17%, (as you mentioned) and see if they align better with the new data?

**Comment by dkobak** [comment]
Okay, it's a mess. For consistency reasons, maybe we should wait until we obtain monthly data for 2020 that agrees with the yearbook yearly sum, and only after that stop using quarterly reports.

**Comment by Nerdm** [comment]
@akarlinsky I was wondering if you could send me the number of expected deaths for 2020 and 2021, based on monthly deaths, based on data from the statistical yearbook (2015-2019). I do not know how to calculate those based on your mathematical model. 

If you could send them on my email addresses dreni.91@gmail.com, it would be immensely helpful.

**Comment by dkobak** [comment]
@Nerdm You can find it in https://github.com/dkobak/excess-mortality/blob/main/baselines.csv

**Comment by akarlinsky** [comment]
Hopefully we'll have 2020 monthly counts at the start of October when ASK releases "Health Statistics 2020".
![image](https://user-images.githubusercontent.com/48435065/135087695-a61047cd-7c80-4c1c-aab6-9f36420723b9.png)

**Comment by akarlinsky** [comment]
"Health Statistics 2020" contained no useful information for us unfortunately.

**Comment by akarlinsky** [comment]
ASK Just released a new version of their quarterly bulletin, and it's a doozy:
https://ask.rks-gov.net/media/6403/buletini-tremujor-tm3-2021-tetori.pdf

2020 and previous monthly values are same as they were.

2021 data now up to September with big revisions for months we already had. It seems that they have moved to the registry numbers rather than survey (https://github.com/akarlinsky/world_mortality/issues/14#issuecomment-918021670) even in the quarterly bulletin. 

This is what we have now:
![image](https://user-images.githubusercontent.com/48435065/139529688-b49315c7-ace3-405a-acd7-29e40fa4a79b.png)

This is the new data:
![image](https://user-images.githubusercontent.com/48435065/139529692-d4123ac1-756c-461b-b030-2d57956c2375.png)

The new data essentially the civil registry data, but total (no separation for inside/outside kosovo). 

For data up to August we can at least remove the deaths that occurred outside of Kosovo, September info will probably be updated on pxstat soon enough:
![image](https://user-images.githubusercontent.com/48435065/139529801-d739d64b-f8d4-4eed-bb2c-137df818fd42.png)

This sort of forces our hand to start using the new registry based monthly data, because now it's all we got and in the same publication. Another option is to postpone updates to Kosovo until we at least get some registry/yearbook data for monthly 2020.

Any thoughts?

**Comment by dkobak** [comment]
I agree that it's time we switch for the registry-based monthly data, only taking deaths INSIDE Kosovo.

I would update 2015-2019 accordingly (from the yearly book), take 2021 values from the table generator for as long as they exist (currently until August), and 

* either leave 2020 data as is (we know it's underestimation, but we can ignore it for now -- the difference is not that big)
* or multiply all 2020 data by a coefficient needed to get the 2020 total equal to 12971 (value shown in the yearly book).

**Comment by Nerdm** [comment]
I agree with @dkobak approach and for 2020 I would suggest using the second option, to get the total right. In this way the seasonal changes would be better aligned.

The data for September should be released on Nov 15, based on previous releases from ASK.

**Comment by dkobak** [comment]
I am closing this as WMD has been updated accordingly.

Thanks @Nerdm.

**Comment by Nerdm** [comment]
Pleased to give my contribution and thank you for the work you've done!

**Comment by akarlinsky** [comment]
From @Nerdm: 
```
ASK (National Statistics Office) published revised data for deaths in 2021.
Also, I wanted to suggest that maybe it would be better to include all deaths, 
minus the deaths of people who reside permanently outside of Kosovo, and not those who died outside of Kosovo, 
since a lot of people who live here seek medical care outside of Kosovo, 
mainly in private hospitals in Macedonia and Turkey.
```

The new numbers are here: 
https://ask.rks-gov.net/media/6478/statistikat-e-lindjeve-vdekjeve-kuror%C3%ABzimeve-dhe-shuror%C3%ABzimeve-shtator-2021.pdf

and table generator here: 
https://askdata.rks-gov.net/PXWeb/pxweb/en/askdata/askdata__09%20Population__Demography%20and%20Migration__Monthly%20indicators__Births,%20deaths,%20marriages%20and%20divorces/tab03.px/?rxid=e41ecc1e-f50c-44f5-be89-091f5d66beae

Our current 2021 data is:
![image](https://user-images.githubusercontent.com/48435065/144565945-c2331a4f-49fb-4272-bfbd-4fa8ceae3272.png)

Revised data (including those dying outside Kosovo) is:
![image](https://user-images.githubusercontent.com/48435065/144565987-be195283-31fa-43d4-888e-dff2cabf5a92.png)

@Nerdm - Any explanation given by ASK for this very big downward revision, especially September?

**Comment by Nerdm** [comment]
@akarlinsky I don't know the reasons why they did that, I just looked into it yesterday and stumbled upon it. I'll look into it.
To be honest, the previous September data looked way over inflated to me personally, given that the hospitalizations were more or less the same as in the previous wave.
Also, the number of testing done during the Delta wave was way higher than previous waves, which should correspond, at least theoretically, in a lower p-score.

**Comment by dkobak** [comment]
This looks weird, but I guess we have no choice but to update the data based on the latest available official information.

Specifically for September 2021, the value we had before (2000) was very round so perhaps was only an estimate.

Regarding deaths outside of Kosovo, I have no opinion, but we should be consistent, and use the same methodology for all years. Using only deaths inside Kosovo (what we had before) makes sense to me, and seems to agree with the yearly tables such as e.g. shown in https://github.com/akarlinsky/world_mortality/issues/14#issuecomment-878191017.

**Comment by akarlinsky** [comment]
Changed to revised data from ASK in commit c82ce812d9d30e30404486479774e138ef361790.
Closing it for now.

**Comment by akarlinsky** [comment]
Latest commit (https://github.com/akarlinsky/world_mortality/commit/fa3a7a05ea90ec05e0b9a6238662db604837ce14) Updated 2020 data to new monthly counts from the 2020 death stats report by ASK.
All counts are now deaths that occurred within Kosovo for internal consistency.

See README for details:
https://github.com/akarlinsky/world_mortality#kosovo-monthly

Keeping this open for a few days for public record.

**Comment by dkobak** [comment]
Okay so does the sum of 2020 now equal 12971 as shown in the table in one of the first posts in this discussion?

And why did 2021 values substantially decrease in the last commit?

**Comment by akarlinsky** [comment]
12987 to be exact. Some minor revisions I guess.

2021 decreased because we now only include deaths that occur in Kosovo, to be consistent with 2020 and 2015-2019.
